sdk/keyvault/keyvault/samples/CMakeLists.txt

```diff
@@ -9,7 +9,8 @@ set(CMAKE_C_STANDARD 99)
 
 # make keys client example
 add_executable (keys_client_example src/keys_client_example.c)
-target_link_libraries(keys_client_example PRIVATE az_keyvault az_core az_core_internal az_curl)
+
+target_link_libraries(keys_client_example PRIVATE az_keyvault az_core az_core_internal)
```

Comment:
OK, I think this just doesn't have the changes I made with Victor. Victor will fix this with his PR

Reply:
My understanding is that we still need this - core is public headers, and you need core_internal whenever you need to use internal headers in your code.